docs/language_specification/statements/instructions/unitary_instructions.md

```diff
@@ -0,0 +1,193 @@
+Unitary instructions, commonly known as gates, define single-qubit or multi-qubit operations
+that change the state of a qubit register in a reversible and deterministic fashion. They include:
+
+- [named gates](unitary_instructions.md#named-gates), or
+- compositions of [gate modifiers](unitary_instructions.md#gate-modifiers) acting on a named gate.
+
+Note that a composition of gate modifiers acting on a named gate is itself a gate. 
+
+??? info "Grammar for unitary instructions"
+    
+    _unitary-instruction_:  
+    &emsp; _gate_ _qubit-arguments_  
+
+    _gate_:  
+    &emsp; _inv-gate_  
+    &emsp; _pow-gate_  
+    &emsp; _ctrl-gate_  
+    &emsp; _named-gate_  
+
+    _inv-gate_:  
+    &emsp; `inv.`gate
+    
+    _pow-gate_:  
+    &emsp; `pow(`_parameter_`).`_gate_
+    
+    _ctrl-gate_:  
+    &emsp; `ctrl.`_gate_
+
+    _named-gate_:  
+    &emsp; _identifier_ _parameters_~opt~  
+
+    _parameters_:  
+    &emsp; __`(`__ _parameter-list_ __`)`__
+
+    _parameter-list_:  
+    &emsp; _parameter_  
+    &emsp; _parameter-list_ __`,`__ _parameter_
+
+    _parameter_:  
+    &emsp; _integer-literal_  
+    &emsp; _floating-literal_
+
+    _qubit-arguments_:  
+    &emsp; _qubit-argument-list_
+
+    _qubit-argument-list_:  
+    &emsp; _qubit-argument_  
+    &emsp; _qubit-argument-list_ __`,`__ _qubit-argument_
+
+    _qubit-argument_:  
+    &emsp; _qubit-variable_  
+    &emsp; _qubit-index_
+
+    _qubit-variable_:  
+    &emsp; _identifier_
+
+    _qubit-index_:  
+    &emsp; _index_
+
+## Named gates
+
+A named gate is what is commonly referred to simply as a gate.
+We use the term named gate to distinguish them from compositions of gate modifiers acting on a named gate,
+which also results to be a gate.
+Named gates are, thus, the simplest form of unitary instructions.
+They can just be used on their own, or modified multiple times by gate modifiers. 
```

Comment:
```suggestion
Named gates comprise particular unitary operations that have been given their own unique label, _e.g._, 
the Hadamard gate **`H`** or the controlled-not gate **`CNOT`**.
In general, we simply refer to them as gates.
All recognized (named) gates are listed in the [Standard gate set](unitary_instructions.md#standard-gate-set).
Moreover, we use the term 'named gate' to distinguish them from unitary operations consisting of compositions of gate modifiers acting on named gates, _i.e._, modified gates.
Named gates are, thus, unmodified gates and can simply be used on their own or modified (multiple times) through gate modifiers. 
```

Reply:
Done.
- I only changed `acting on named gates` to `acting on a named gate` because the same text is used in 3 other places in the doc.
- And I've removed the line `The named gates that are supported by the cQASM language
are listed in the [standard gate set](unitary_instructions.md#standard-gate-set).` a bit further down in the section, because it's basically a duplicate of the line 65 above.